vulcan/lib/server/workflows/scripts/dge_filter.py

```diff
@@ -21,12 +21,14 @@
 # min_pct
 min_pct = float(input_var('min_pct'))
 # Algorithm choice: OR filter, a.k.a. either group meets the cutoff
-pct_cols = [s for s in DEdf.keys() if (s=="pts" or s.startswith("pts_"))]
-pct_col1 = list(DEdf[pct_cols[0]])
-pct_col2 = list(DEdf[pct_cols[1]])
-idx_passed = [
-    i for i in list(DEdf_filt.index) if ( pct_col1[i] >= min_pct or pct_col2[i] >= min_pct )]
-DEdf_filt = DEdf_filt.loc[idx_passed,:]
```

Comment:
Instead of decreasing the number of rows, this often led to a massive increase!  Pandas is confusing and I'm not sure exactly how this worked... but fix below